### ISSUE TITLE
Acceptance tests are not compatible with cf cli v7

### ISSUE THREAD

**Issue by wayneadams** [cloudfoundry/cf-networking-release#82]
## Issue

Tests within [src/test/acceptance](https://github.com/cloudfoundry/cf-networking-release/tree/develop/src/test/acceptance) are not compatible with cf cli v7

## Context

While running integration tests, we found that the `cf_cli_adapter` does not handle the cf cli v7 api for the ` bind-security-group` command. 

## Steps to Reproduce

Run the acceptance set suite using cf cli v7

## Expected result

All tests pass.

## Current result

```
STEP: removing test-generated ASGs
• Failure [70.704 seconds]
external connectivity
/tmp/build/29505db7/acceptance-tests/src/test/acceptance/external_connectivity_test.go:20
  basic (legacy) network behavior for an app
  /tmp/build/29505db7/acceptance-tests/src/test/acceptance/external_connectivity_test.go:108
    allows outbound ICMP only if allowed [It]
    /tmp/build/29505db7/acceptance-tests/src/test/acceptance/external_connectivity_test.go:147

    Expected success, but got an error:
        <*cf_cli_adapter.CmdErr | 0xc0004d8870>: {
            Out: "FAILED\n\nNAME:\n   bind-security-group - Bind a security group to a particular space, or all existing spaces of an org\n\nUSAGE:\n   cf bind-security-group SECURITY_GROUP ORG [--lifecycle (running | staging)] [--space SPACE]\n\nTIP: Changes require an app restart (for running) or restage (for staging) to apply to existing applications.\n\nOPTIONS:\n   --lifecycle      Lifecycle phase the group applies to. (Default: running)\n   --space          Space to bind the security group to. (Default: all existing spaces in org)\n\nSEE ALSO:\n   apps, bind-running-security-group, bind-staging-security-group, restart, security-groups\n",
            Err: "Incorrect Usage: unexpected argument \"test-space\"\n",
            Message: "exit status 1",
        }
        exit status 1:
        
        Out:
        FAILED
        
        NAME:
           bind-security-group - Bind a security group to a particular space, or all existing spaces of an org
        
        USAGE:
           cf bind-security-group SECURITY_GROUP ORG [--lifecycle (running | staging)] [--space SPACE]
        
        TIP: Changes require an app restart (for running) or restage (for staging) to apply to existing applications.
        
        OPTIONS:
           --lifecycle      Lifecycle phase the group applies to. (Default: running)
           --space          Space to bind the security group to. (Default: all existing spaces in org)
        
        SEE ALSO:
           apps, bind-running-security-group, bind-staging-security-group, restart, security-groups
        
        
        Err:Incorrect Usage: unexpected argument "test-space"
        
        

    /tmp/build/29505db7/acceptance-tests/src/test/acceptance/external_connectivity_test.go:156
```

## Possible Fix

Update the `cf_cli_adapter` method `BindSecurityGroup` ([here](https://github.com/cloudfoundry/cf-networking-release/blob/f375bd37a386ff52afc4faae1081b08f0014eaef/src/cf-pusher/cf_cli_adapter/adapter.go#L204)) to handle cf cli v7.

**Comment by mcwumbly** [comment]
We'd love to get this addressed and will do so shortly, but would welcome a PR if someone wants to raise their hand and grab it first :)

**Comment by jrussett** [comment]
Hi @wayneadams 👋 

We've made the requisite changes to the acceptance tests in this commit: https://github.com/cloudfoundry/cf-networking-release/commit/6f350ca6bebfc095e6468119abf1c1206de21d0b

We are now going to go through the process of finalizing new releases so y'all can consume them in CI. 

Thanks, 
@jrussett

**Comment by jrussett** [comment]
The fix has been released in [`cf-networking-release` 2.32.0](https://github.com/cloudfoundry/cf-networking-release/releases/tag/2.32.0). 🥳

**Comment by wayneadams** [comment]
Nice @jrussett! Closing this issue.
